### ISSUE TITLE
Missing Reload

### ISSUE THREAD

**Issue by KK20482** [eirannejad/pyRevit#951]
Everytime I start Revit I get this error message:
﻿﻿WARNING [pyrevit.loader.systemdiag] You are not using the required host build: 0
I'd like to not see this please. Thank you.

![image](https://user-images.githubusercontent.com/69061351/89088041-43b27c00-d34b-11ea-8c87-8399b686297b.png)

**Comment by eirannejad** [comment]
You are using an old beta version of pyRevit. Please install the latest from link below and try
https://github.com/eirannejad/pyRevit/releases

**Comment by KK20482** [comment]
Hi Ehsan,
I don’t have permission to run installer. So I ran our company installer which is supposed to update pyRevit. It showed the following message:

>>>[pyRevit version 4.6.11] is installed & updated by a newer version of the Revit Install/Update package!

And when I ran Revit, I received following messages during Revit start-up.

[cid:image005.png@01D66983.942519E0]
And this message.
[cid:image006.png@01D66983.942519E0]

Can I manually update pyRevit instead?
Thank you.


KEVIN KUANG
HOK
kevin.kuang@hok.com
t +1 415 356 8779
